compass/ocean/tests/global_ocean/files_for_e3sm/add_total_iceberg_ice_shelf_melt.py

```diff
@@ -0,0 +1,134 @@
+import os
+
+import numpy as np
+import xarray as xr
+from mpas_tools.io import write_netcdf
+
+from compass.io import symlink
+from compass.ocean.tests.global_ocean.files_for_e3sm.files_for_e3sm_step import (  # noqa: E501
+    FilesForE3SMStep,
+)
+
+
+class AddTotalIcebergIceShelfMelt(FilesForE3SMStep):
```

Comment:
On second thought, I think generalizing Compass so it can handle new iceberg and ISMF datasets based on config options seems like a fairly significant challenge that's beyond the scope of this PR.  I think it would make sense to take that on as soon as we have such a new dataset.